src/Data/Masa.BuildingBlocks.Data.UoW/IUnitOfWorkAccessor.cs

```diff
@@ -9,4 +9,6 @@ public interface IUnitOfWorkAccessor
     /// Only exists after the DbContext is confirmed to be created
     /// </summary>
     MasaDbContextConfigurationOptions? CurrentDbContextOptions { get; set; }
+
+    IDisposable SetTemporaryCurrentDbContextOptions(MasaDbContextConfigurationOptions options);
```

Comment:
What's the mean of temporary?

Reply:
Used to temporarily change the current DbContext link information, used to obtain the host's DbContext string for distributed locks